### DISCUSSION TITLE
Support for Quarto LaTeX pdfs?

### DISCUSSION THREAD

**Discussion by jimgar** [posit-dev/brand-yml#43] [Ideas]
Hi!

Super cool idea to have a standard file that can be used across a bunch of tools!

Just out of curiosity, because I know it is still widely used, are there plans to support pdfs made with LaTeX?

I noticed [on the site](https://posit-dev.github.io/brand-yml/) that for Quarto there is support for pdfs made with Typst. But LaTeX isn't in the diagram and greyed out:
 
<img width="809" alt="Screenshot 2024-10-29 at 13 41 23" src="https://github.com/user-attachments/assets/1289f97a-8bcf-43b4-926f-e4f27d524066">

Cheers
Jim

**Comment by gadenbuie** [comment]
> Super cool idea to have a standard file that can be used across a bunch of tools!

Thanks!

> Just out of curiosity, because I know it is still widely used, are there plans to support pdfs made with LaTeX?

LaTeX isn't part of the initial Quarto support for brand.yml, unfortunately, but it's not technically impossible. This diagram broadly represents my understanding of our collective plans, but it's not completely up-to-date with the Quarto team's road map. So, as I understand it, LaTeX isn't part of the immediate plans but that could definitely change depending on brand.yml usage, community response, etc.

**Reply by jimgar** [reply]
Nice one @gadenbuie.

Any idea who would be best for me to ask in the Quarto team?
